### PR TITLE
CircleCI Nightly Scheduled Trigger

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23265

## Description
Schedules a nightly build to be triggered using the CircleCI `config.yml` itself. Also, disabled the pre-existing crontab job.

## Minimum version of Demisto
- [x] 4.5.0
- [x] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by avidan-H** [comment]
Adding `docs-approved` label since there is no documentation to be reviewed.

File: .circleci/config.yml [hunk 1]
```diff
@@ -604,4 +604,16 @@ workflows:
...
     jobs:
       - build:
-          context: instances_test_env
\ No newline at end of file
+          context: instances_test_env
+  nightly:
+    triggers:
+      - schedule:
+          # should trigger every day at 9 PM UTC (12 AM Israel Time)
+          cron: "0 21 * * *"
```
Comment (yaakovi) [inline .circleci/config.yml:612]: should be trigger at 3:00 or 4:00 (AMI creation is at 1:00)
and we want the maximum amount of time during the day before the machines are down (as long as the build finishes before people start working (around 8:00-9:00).

@ShahafBenYakir WDYT?

Comment (ShahafBenYakir) [reply .circleci/config.yml:612]: cool

Comment (yaakovi) [reply .circleci/config.yml:612]:
```suggestion
          cron: "0 0 * * *"
```

**Pull request merged**